deploy/docker/notebook/stacks/base_gpu.yaml

```diff
@@ -0,0 +1,406 @@
+name: base_gpu
+base: "ktaletsk/gpu:bionic-20190612-cuda-10-1@sha256:010bda5d55ddfafd4b24b9a452d4539b07745441cdafe7ef6fdfea9b13396ec9"
```

Comment:
Is this Dockerfile versioned controlled on GitHub somewhere? Can we at least add the GitHub link in a comment or move this into LabShare somewhere?

Reply:
Source Dockerfile is currently here: https://github.com/ktaletsk/gpu_images
But I would prefer to move it to `Labshare` org

Reply:
Manually pushed `labshare/polus-cuda-base:bionic-20190612-cuda-10-1` image, will create autobuild later, if new repo is created in `Labshare`.